vulnerabilities/tests/test_data/archlinux.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "name": "AVG-837",
```

Comment:
If you were to explain this record  in plain english, what would say? How many vulnerabilities references do we have in there? I think you are missing out all the CVE references and the fact these are arch linux packages

Reply:
ping?